pkgs/by-name/km/kmax/package.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, python3Packages
+}:
+
+let
+  pname = "kmax";
+  version = "4.5.2";
+  src = fetchFromGitHub {
+    owner = "paulgazz";
+    repo = "kmax";
+    rev = "v${version}";
+    hash = "sha256-9LAR+1MdT2YjsPAvmSIdiBTqUh5WYdJuREzLYHRMIMw=";
+  };
+in
+python3Packages.buildPythonPackage {
+  inherit pname version src;
+
+  propagatedBuildInputs = with python3Packages; [
+    packaging
+    regex
+    requests
+    setuptools
+    whatthepatch
+    z3-solver
+  ];
+
+  nativeCheckInputs = with python3Packages; [
+    pytestCheckHook
+  ];
+
+  meta = {
+    homepage = "https://github.com/paulgazz/kmax";
+    description = "Linux Kconfig constraint SAT/SMT solver";
+    license = with lib.licenses; [ gpl2Plus ];
+    mainProgram = "kmax";
+    maintainers = with lib.maintainers; [ AndersonTorres ];
+    sourceProvenance = with lib.sourceTypes; [ fromSource ];
```

Comment:
The problem is:

> A package with no meta.sourceProvenance set implies it has no known sourceTypes other than fromSource.

And we know that all sources from this particular package are Python source files.

In this sense, I interpret this as "if fromSource is present, then we are sure that everything is human-readable  source files; else, it can be a mix of source, binaries, bytecodes etc.".